docs/agent_hocon_reference.md

```diff
@@ -139,6 +140,11 @@ The agent_llm_info_file key allows you to specify a custom HOCON file that exten
 
 For more information on selecting and customizing models, see the [model_name](#model_name) section below.
 
+### toolbox_info_file
+
+The agent_toolbox_info_file key lets you define a custom HOCON file that adds to the default set of tools available to agents within a neuro-san network. This is particularly helpful when you have tools shared across multiple agent networks.
+
+For further details, refer to the [toolbox](#toolbox) section below.
```

Comment:
Perhaps also worth providing a link to the toolbox_info_hocon_reference.md file as well.